env0/utils.go

```diff
@@ -50,6 +50,29 @@ func stringInSlice(str string, strs []string) bool {
 	return false
 }
 
+func reasourceDataGetValue(fieldName string, d *schema.ResourceData) interface{} {
+	dval := d.Get(fieldName)
+	if dval == nil {
+		return nil
+	}
+
+	_, okInt := dval.(int)
+	_, okBool := dval.(bool)
+	_, okString := dval.(string)
+	if okString || okBool || okInt {
+		//lint:ignore SA1019 reason: https://github.com/hashicorp/terraform-plugin-sdk/issues/817
+		if _, exists := d.GetOkExists(fieldName); !exists {
+			return nil
+		}
+	}
+
+	if sval, ok := dval.([]interface{}); ok && len(sval) == 0 {
```

Comment:
If a "list" is empty, no need to write an empty list to the interface. `nil` is more correct.